draft-ietf-ppm-dap.md

```diff
@@ -390,6 +390,7 @@ in the "type" field (within the DAP URN namespace
 | batchLifetimeExceeded      | The batch lifetime has been exceeded for one or more reports included in the batch. |
 | batchMismatch              | Aggregators disagree on the report shares that were aggregated in a batch. |
 | unauthorizedRequest        | Authentication of an HTTP request failed (see {{https-sender-auth}}). |
+| missingTaskId              | `/hpke_config` endpoint received a request without `task_id` parameter (see {{hpke-config}}). |
```

Comment:
While precise, this description of error is hard to parse without knowing the protocol context. What's going on here, intuitively?

Reply:
Also, @tgeoghegan: would you prefer `missingTaskId` or `missingTaskID`? I think the latter might be more consistent.

Reply:
> While precise, this description of error is hard to parse without knowing the protocol context. What's going on here, intuitively?

Should I add on to the above text with "if a deployment enforces different HPKE configurations for different tasks"?

Regarding TaskId vs TaskID, I think the type definition is `TaskId` at line 608, but there are struct fields where `TaskID` is used, so it seems like there is some inconsistency here. I was trying to follow the original type definition.

Reply:
> Should I add on to the above text with "if a deployment enforces different HPKE configurations for different tasks"?

I would simplify it. How about:
```
HPKE config requested without specifying the task ID
```

> Regarding TaskId vs TaskID, I think the type definition is `TaskId` at line 608, but there are struct fields where `TaskID` is used, so it seems like there is some inconsistency here. I was trying to follow the original type definition.

Ah OK, nice catch. Let's correct the inconsistency here. It seems like the preference is likely to be "ID".

Reply:
[RFC 7807](https://www.rfc-editor.org/rfc/rfc7807.html#section-4) doesn't have any obvious guidance I can see, but if we look at [ACME aka RFC 8555](https://www.rfc-editor.org/rfc/rfc8555.html#section-6.7)'s errors, it has types like `badCSR`, `caa` and `dns`, which I think would lead us to using `missingTaskId`, and that would be consistent with the struct definitions elsewhere. But I'm happy with either; the ambiguous case problems weren't introduced by this change and can always be cleaned up later in the standardization process.

Reply:
Works for me